source/_components/logi_circle.markdown

```diff
@@ -8,12 +8,36 @@ comments: false
 sharing: true
 footer: true
 logo: logi_circle.png
-ha_category: Camera
+ha_category:
+  - Camera
+  - Sensor
 ha_release: 0.79
 ha_iot_class: Cloud Polling
+redirect_from:
+  - /components/camera.logi_circle/
+  - /components/sensor.logi_circle/
+  - /components/logi_circle.camera/
```

Comment:
Have these pages existed previously? The address isn't correct.

```
/components/logi_circle.camera/
/components/logi_circle.sensor/
```

Reply:
They didn't. I only added them to match the import path of the component, but happy to get rid of it if it's not appropriate.

Reply:
If you remove them then we can perform the merge 😉

Reply:
@klaasnicolaas Done.